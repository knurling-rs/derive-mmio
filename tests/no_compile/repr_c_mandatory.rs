#[derive(derive_mmio::Mmio)]
struct Uart {
    data: u32,
}

fn main() {}
