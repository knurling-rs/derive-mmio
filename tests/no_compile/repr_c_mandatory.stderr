error: `#[derive(Mmio)]` only works on repr(C) types
 --> tests/no_compile/repr_c_mandatory.rs:1:10
  |
1 | #[derive(derive_mmio::Mmio)]
  |          ^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the derive macro `derive_mmio::Mmio` (in Nightly builds, run with -Z macro-backtrace for more info)
